### PR TITLE
WIP: add x86 image documentation, close #710

### LINKED ISSUE #710: x86 image documentation
Now that we have the RPD ISO on the [downloads page](https://www.raspberrypi.org/downloads/raspberry-pi-desktop/), it would be good to be able to link to some documentation on how to use it. I propose:

- How to write the image to a CD/DVD/USB stick
- How to boot from a live disc
- How to install it on a PC
- How to boot it in a VM

I'd be happy to write the VM tutorial, and I know we've had magpi articles on writing to a USB stick, but we should place them in here and keep them up-to-date. I haven't seen any tutorials on how to install it for good - I know there are some new options on boot in the more recent image, and these good do with explaining too.

Any volunteers?

### PR DESCRIPTION
**Pull request by bennuttall**
Initial commit of RPD image docs - more work to do, and will need some help with the content

### CODE REVIEW

File: installation/raspberry-pi-desktop/README.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
+# Raspberry Pi Desktop for PC and Mac
+
+The Raspberry Pi Foundation provides a Debian-based operating system image
+for x86 computers with the Raspberry Pi Desktop skin found in Raspbian.
+This image contains most of the programming tools and software found in Raspbian
+and is intended to make these tools available to people without a Raspberry Pi.
+
+Users can install the image on a CD/DVD or USB stick, and either use it as a
+live disc (boot the OS without Installing it), install it on their computer or
```
Comment (lurch) [inline installation/raspberry-pi-desktop/README.md]: `Installing` -> `installing`

File: installation/raspberry-pi-desktop/README.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
+# Raspberry Pi Desktop for PC and Mac
+
+The Raspberry Pi Foundation provides a Debian-based operating system image
+for x86 computers with the Raspberry Pi Desktop skin found in Raspbian.
```
Comment (lurch) [inline installation/raspberry-pi-desktop/README.md]: Might be worth mentioning that it boots on x64 computers too? (or is that too much of a subtle technical difference that most people won't understand?)

**Comment by lurch** [comment]
I guess most of the "installing onto a USB stick" is already (indirectly) covered by https://github.com/raspberrypi/documentation/tree/master/installation/installing-images ?

File: installation/raspberry-pi-desktop/README.md (hunk 2 above)
Comment (bennuttall) [inline installation/raspberry-pi-desktop/README.md]: Thanks - I'll reword

**Comment by bennuttall** [comment]
I've added some content - I'll be away for 2 weeks so if anyone is able to fill in the gaps, that would be handy.

**Comment by lurch** [comment]
There's a couple of places where "Insert the disk or USB stick in your computer before or after it starts booting." is used. As the 'disk' in this context is a DVD, I guess this should be 'disc'? Also, should the "before or after it starts booting" be changed to just "before it starts booting"? Not much point inserting the disc or USB stick _after_ the computer has started booting?!

Also, there's a couple of places where the phrase "a CD/DVD" is used - the ISO image is far too big to fit on a CD, and will only fit on a DVD.

**Comment by JamesH65** [comment]
@bennuttall Since you are back, have you got time to go through and finish this one off so it can be flagged for copy edit?

**Comment by bennuttall** [comment]
I was looking to get some comments on what people think

**Comment by JamesH65** [comment]
AFAICT, it only needs the installing on the USB stick section completed and that should be good enough to send to copy edit. We can always adjust/add later as necessary.

**Comment by JamesH65** [comment]
This seems to have stalled - is anyone able to fill out the USB section?

**Comment by JamesH65** [comment]
@bennuttall @lurch Should we just commit as is and fill in the missing sections at a later date?

**Comment by bennuttall** [comment]
Sorry @JamesH65 I'm back (again) now. I'll try and tidy up to get ready to go live and we can add more detail later.

**Comment by JamesH65** [comment]
@bennuttall Hi Ben, what is left to do on this? I've got some spare time at home today so can full in any gaps - is it just the creating of the USB stick section as mentioned above?

**Comment by JamesH65** [comment]
@bennuttall @lurch I've added the USB/DVD section, and removed references to CD's - current image is too large for a CD. Anything else need to be added?

**Comment by JamesH65** [comment]
Is this ready to go to copy edit? Be nice to get it merged. Ping @bennuttall @lurch

**Comment by bennuttall** [comment]
Thanks @JamesH65 - I'm going to make a few small edits and it should be good to go. I've a feeling the screenshots might be out-of-date if they've not been updated since I first pushed them, but I can redo them easily enough.

**Comment by JamesH65** [comment]
I'll flag as ready for CE then, that won't affect the pictures, and it'll probably be a while before the CE is done anyway.

**Comment by bennuttall** [comment]
Pushed some more changes and added new screenshots.

@XECDesign would you be able to write a short bullet for each of the advanced boot options to clarify what they do here: https://github.com/raspberrypi/documentation/blob/710-x86-image-docs/installation/raspberry-pi-desktop/boot-options/README.md#options

Also I assume persistence is only available when on a USB stick (or VM) and not when on a DVD. Are the boot options different then, and/or should we point this out?

**Comment by XECDesign** [comment]
Expert mode shows all the install options - may result in a standard debian install rather than one with RPD.
Rescue mode gives you access to tools to recover a failed install (if you need to reinstall the bootloader or access to a shell, for example).
Automated mode uses a user-provided preseed file which probably won't work, since we already provide one.

The advanced menu probably needs a warning rather than a description of the options.

File: installation/raspberry-pi-desktop/boot-options/README.md [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+1. **Run and reset persistence**
+    - Reset any changes made, and live boot the OS with persistence of files
+    thereafter
+1. **Run without persistence**
+    - Run the OS with no persistence of changes
+1. **Install**
+    - Install the OS fully using a command-line installer
+1. **Graphical install**
+    - Install the OS fully using a graphical installer
+1. **Advanced options**
```
Comment (JaninaPi) [inline installation/raspberry-pi-desktop/boot-options/README.md]: Are there plans to replace the `???`?

Comment (JamesH65) [reply installation/raspberry-pi-desktop/boot-options/README.md]: Hmmm, hadn't realised there were sections missing.  @bennuttall Should we remove until have the required text, or fill in now?

**Comment by JaninaPi** [comment]
Have checked all the files, will leave you to sort out the missing content.

**Comment by JamesH65** [comment]
Thanks!

**Comment by bennuttall** [comment]
TODO: add link from installing page!

**Comment by JamesH65** [comment]
We should probably resurrect this...

**Comment by bennuttall** [comment]
Yup. There's a guide here now too: https://projects.raspberrypi.org/en/projects/install-raspberry-pi-desktop

**Comment by bennuttall** [comment]
Perhaps what we need here is documentation about it rather than a tutorial?

**Comment by aallan** [comment]
In light of #1911 is this PR still relevant? If there is supporting third-party documentation outside of this repo, and there hasn't been much recent progress I'd be inclined to close it?

**Pull request closed without merging**